docs/integrations/gitlab/locations.md

```diff
@@ -21,6 +21,18 @@ integrations:
 
 :::note Note
 
+When using the [`backstage.io/source-location`](../../features/software-catalog/well-known-annotations.md#backstageiosource-location): `url:https://{gitlab host}/{group}/{subgroup}/{repository}/` annotation to create a component, you must include `/blob/main/` in the URL path. For example:
```

Comment:
Heya! 👋 

I was just looking at https://github.com/backstage/backstage/issues/28981 I am not sure what the `README` part refers to, I assume maybe its a custom plugin maybe 😅 

But I think the message should be here that if you want to manually set this annotation then it must contain the `/blob/main/` 

AFAIK, If you import something from Gitlab into the catalog via the UI it automatically adds this annotation so I assume in that issue its being manually set for some reason or other.

Sorry for taking so long to look into this also! 😞

Reply:
hello, I believe the readme section refers to readme but since the link is wrong it will fail to load

Reply:
Hey! 👋 

What I am trying to say is that, I think the plugin (the part that displays the README) is a custom plugin, but correct me if I am wrong!

So I'm worried people who don't use this plugin might read it and get confused "What do you mean the README wont display? What README?" 😅, so I guess the aim of the message should be that if you are setting the annotation yourself (manually) then make sure you include the `/blob/main/` part as its needed, what do you think? Or am I way off here 😆